zum/requests/core.py

```diff
@@ -2,12 +2,44 @@
 Module for the core requests logic of zum.
 """
 
-from typing import Any, Dict, List
+from json import load
+from pathlib import Path
+from typing import Any, Dict, List, NoReturn, Union
 
+from zum.requests.errors import InvalidRequestBodyFileError
 from zum.requests.helpers import reduce_arguments
 from zum.requests.models import Request
 
 
+def load_request_body(arguments: List[str]) -> Union[Dict[str, Any], NoReturn]:
```

Comment:
I think that this method could explicitly say `file`, so that it is obvious that it retrieves the contents of a file. For example, it could be named `load_request_body_from_file`. Also, I think that it should receive only the first element of the `arguments` list (pre-processing that information outside this method). I don't know what should happen if `arguments` were to be empty, maybe you should pass `None` or directly raise the error from outside the method. I leave that to your judgement. Lastly, the return type should be only `Dict[str, Any]`. On another comment I left I explained how to stop `mypy` from whining about the return type by refactoring the method's code